app/views/providers/home_address/home_address_selections/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_with(
+      model: @form,
+      scope: :address_selection,
+      url: providers_legal_aid_application_home_address_home_address_selection_path(@legal_aid_application),
+      method: :patch,
+      local: true,
+    ) do |form| %>
+
+    <% if @address_collection.count > 1 %>
```

Comment:
Improvement: may be possible to use `.many?` instead of `.count > 1`

Reply:
have made suggested change